packages/wonder-stuff-i18n/src/utils/emoji-for-locale.test.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import {getEmojiForLocale} from "./emoji-for-locale.js";
+
+describe("getEmojiForLocale", () => {
+    it("returns an emoji for boxes", () => {
+        // Arrange
+        const locale = "boxes";
+
+        // Act
+        const result = getEmojiForLocale(locale);
+
+        // Assert
+        expect(result).toEqual("📦");
+    });
+
+    it("returns an emoji for es", () => {
+        // Arrange
+        const locale = "es";
+
+        // Act
+        const result = getEmojiForLocale(locale);
+
+        // Assert
+        expect(result).toEqual("🇲🇽");
+    });
+
+    it("handles an unknown locale", () => {
+        // Arrange
+        const locale = "asdf";
+
+        // Act
+        const result = getEmojiForLocale(locale);
+
+        // Assert
+        expect(result).toEqual("");
+    });
```

Comment:
When we filter out `en` out of the list of locales to translate I wonder if we should also filter out unknown locales as well.

Reply:
Hmm - so right now we don't really have a concept of "known" and "unknown" locales (only in-so-much as when we don't get any strings for a locale). So I'm not entirely sure it makes sense to do that (I'd rather just error out because we can't find the strings).